app/components/ui/connect-user/index.js

```diff
@@ -108,8 +107,8 @@ const ConnectUser = React.createClass( {
 					{ i18n.translate( "Don't have an account yet?" ) }
 				</h3>
 				<p>
-					{ i18n.translate( 'Find a .blog you love to {{link}}get started now{{/link}}.', {
-						components: { link: <Link to={ getPath( 'search' ) } /> }
+					{ i18n.translate( 'Find a .blog you love on {{link}}wordpress.com/domains{{/link}}.', {
```

Comment:
That's how it was in the original version (I mean, without `strong`), so I just based the update on that. At this point, I don't think it will make much difference.